src/Css.elm

```diff
@@ -833,6 +911,366 @@ n =
 
 
 
+{- ANGLES -}
+
+
+angleConverter : String -> number -> Angle
+angleConverter suffix num =
+    ((numberToString num) ++ suffix)
+        |> ExplicitAngle
+        |> ExplicitValue
+
+
+{-| [`deg`](https://developer.mozilla.org/en-US/docs/Web/CSS/angle#deg) units.
+-}
+deg : number -> Angle
+deg =
+    angleConverter "deg"
+
+
+{-| [`grad`](https://developer.mozilla.org/en-US/docs/Web/CSS/angle#grad) units.
+-}
+grad : number -> Angle
+grad =
+    angleConverter "grad"
+
+
+{-| [`rad`](https://developer.mozilla.org/en-US/docs/Web/CSS/angle#rad) units.
+-}
+rad : number -> Angle
+rad =
+    angleConverter "rad"
+
+
+{-| [`turn`](https://developer.mozilla.org/en-US/docs/Web/CSS/angle#tr) units.
+-}
+turn : number -> Angle
+turn =
+    angleConverter "turn"
+
+
+
+{- TRANSFORMS -}
+
+
+cssFunction : String -> (a -> String) -> List a -> String
+cssFunction funcName converter args =
+    args
+        |> List.map converter
+        |> String.join ", "
+        |> (++) (funcName ++ "(")
+        |> (flip (++)) ")"
```

Comment:
Definitely! I initially planned to do `cssFunction`, `cssFunction2`, etc. like you had done with `prop1`, `prop2`, etc. but then I saw that `matrix()` accepts 16 arguments and css functions almost always accept the same data type for all arguments so I figured lifting to a `List` would be cool.